app/src/main.cpp

```diff
@@ -272,10 +285,17 @@ namespace scitt
         });
 
         ctx.rpc_ctx->set_response_status(HTTP_STATUS_CREATED);
+
+        // TODO log CoseProfile
+        SCITT_INFO("ClaimSizeKb={}", body.size() / 1024);
       };
 
-      make_endpoint(
-        post_entry_path, HTTP_POST, error_adapter(post_entry), authn_policy)
+      make_endpoint_with_local_commit_handler(
+        post_entry_path,
+        HTTP_POST,
+        tracing_adapter(error_adapter(post_entry), get_time),
+        tracing_local_commit_callback,
+        authn_policy)
         .install();
```

Comment:
At some point I think we should define our own `make_endpoint` function that provides all the right defaults. Happy to do this in a later PR.

Reply:
Sure, let's do that in the future sometime.